### PR TITLE
Implement ListBoxRowImplExt for activate method

### PR DESCRIPTION
**Pull request by matzipan**

### CODE REVIEW

File: gtk/src/subclass/list_box_row.rs [hunk 1]
```diff
@@ -19,6 +19,21 @@ pub trait ListBoxRowImplExt: ObjectSubclass {
     fn list_box_row_activate(&self, list_box_row: &Self::Type);
 }
 
+impl<T: ListBoxRowImpl> ListBoxRowImplExt for T {
+    fn list_box_row_activate(&self, list_box_row: &Self::Type) {
```
Comment (bilelmoussaoui) [inline gtk/src/subclass/list_box_row.rs:23]: This should be renamed to `parent_activate` to follow the same naming conventions we use everywhere.

Comment (matzipan) [reply gtk/src/subclass/list_box_row.rs:23]: Should this be part of this PR? Or a subsequent one?

Comment (matzipan) [reply gtk/src/subclass/list_box_row.rs]: I've done it here.

Comment (bilelmoussaoui) [reply gtk/src/subclass/list_box_row.rs]: As part of this PR is fine

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**